src/tests/multihost/alltests/test_ldap_time_logging.py

```diff
@@ -0,0 +1,64 @@
+"""Automation for ldap_query_time_logging
+
+:requirement: LDAP Query Time Logging
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import re
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.timelog
+@pytest.mark.tier1_3
+class TestLdapTimeLogging(object):
+    """ Test ldap_time_logging """
+
+    def test_0001_bz1925559(self, multihost, backupsssdconf):
+        """
+        :title: Check time logging is not enabled by default
+        :id: 11379bae-99bd-4dd9-bee6-fa0dba28f4fe
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1925559
+        """
+        section = "domain/%s" % ds_instance_name
+        tools = sssdTools(multihost.client[0])
+        domain_params = {'debug_level': '0'}
+        tools.sssd_conf(section, domain_params, action="delete")
+        tools.clear_sssd_cache()
+
+        user = 'foo9@%s' % ds_instance_name
+        get_user = f'getent passwd {user}'
+        multihost.client[0].run_command(get_user, raiseonerr=False)
+        log_list = ['sssd', f'sssd_{ds_instance_name}',
+                    'sssd_nss', 'sssd_pam']
+        for log in log_list:
+            log = f'/var/log/sssd/{log}.log'
+            log_str = multihost.client[0].get_file_contents(log).decode('utf-8')
+            find = re.compile(r'milliseconds')
+            assert not find.search(log_str)
+
+    def test_0002_bz1925559(self, multihost, backupsssdconf):
+        """
+        :title: Time logging is enabled for debug level 0x20000 above
```

Comment:
I would suggest to remove `above`, `0x20000` is the bitmask to select the timing/statistical data, so there is no 'above' like in the levels 0-9. Besides with `0x20000` the timing data should be only present with `debug_level = 9` and above (there is only 10 left). So I think you can modify the first test by setting `debug_level = 8` and check that there are no messages. Btw, `debug_level = 0` is not the default anymore, best to not set it at all, if you want to check the default.

Reply:
Removed `above` from the title and commit message. First test modified to test with debug_level 8. I forgot to remove `0`, it is actually deleting the param `debug_lelvel` if it exists in the conf.